app/util/cmr.ts

```diff
@@ -453,3 +455,16 @@ export async function belongsToGroup(
   const response = await cmrSearchBase(path, null, token, { 'X-Harmony-User': username });
   return response.status === 200 && (response.data as string[]).indexOf(username) !== -1;
 }
+
+/**
+ * Return all non-inherited links with rel ending in /data# when no regex is provided.
+ *
+ * @param granule - The granule to obtain links from
+ * @returns An array of granule links
+ */
+export function filterGranuleLinks(
+  granule: CmrGranule,
+): CmrGranuleLink[] {
+  return granule.links.filter((g) => (g.rel.endsWith('/data#') || g.rel.endsWith('/service#'))
```

Comment:
It looks like you are only ever. using the first link that matches, in which case you should use `find` here instead of `filter`.

Reply:
I figured I would leave this function as is since it returns all the possible data links. I assume that's what we should be doing, it just so happens that right now this old code ends up only sending one data link in backend service messages. This is only relevant in legacy templates which pass links rather than using the STAC catalog from the query-cmr task. In the new style we do include multiple links in the STAC catalog.